src/commands/aicommits.ts

```diff
@@ -93,6 +94,21 @@ export default async (
 		message = selected;
 	}
 
+	if (editBeforeCommit) {
+		const updatedMessage = await text({
+			message: 'Edit commit message',
+			initialValue: message,
+			placeholder: 'Add a commit message here',
+		});
+
+		if (isCancel(updatedMessage)) {
+			outro('Commit cancelled');
+			return;
+		}
+
+		message = updatedMessage as string;
+	}
+
```

Comment:
Instead of putting this behind an `--edit` flag, let's refactor the confirmation prompt to be:
1. Commit
2. Edit message

Reply:
Hmm that makes more sense. Will do that.

Reply:
Yep!

Reply:
Hmm okey.Makes sense.